src/sora-builders.md

```diff
@@ -25,38 +25,56 @@ The SORA Builders Programme is an extension of the [SORA RFP program](/rfp.md).
 
 Both new projects and ports can receive compensation in the form of XOR, as well as development support from the SORA community. Because effectively, time and development efforts are not free, and in order to uphold the community standards, not everyone who applies will be accepted by the programme.
 
-It is also important to mention that depending on the value added by the project, it may also qualify for [Social Insurance](https://medium.com/sora-xor/social-insurance-for-systematically-important-infrastructure-18a63ef711ca), however, there will be required due diligence and the project will have to prove systematic importance to the community.
+It is also important to mention that depending on the value added by the project, it may also qualify for [Social Insurance](/social-insurance.md), however, there will be required due diligence and the project will have to prove systematic importance to the community.
 
 It is easy to ask _wot_ and _wen_ when it comes to development, but _putting your RFP where your complaints are_ is not that simple to do. The community has no limitation to what is proposed, however, _[spam](https://youtu.be/duFierM1yDg)_, _CEX requests_, and other off-topic requests will not be considered.
 
-**To join the SORA Builders Programme, please [fill this form](https://forms.gle/1cdvKTHLXnBXbMTr9), then join the [SORA Builders Telegram group](https://t.me/sorabuilders)**.
+## The Buidling Process
 
-1.  **Sign up**: After you have submitted your proposal, it will be reviewed
-    internally and you will be invited to talk to the developers, if it
-    is a match, the process will continue.
+To join the SORA Builders Programme, please [fill this form](https://forms.gle/1cdvKTHLXnBXbMTr9), then join the [SORA Builders Telegram group](https://t.me/sorabuilders). 
 
-2.  **Milestones**: Funding support provided by the SORA Builders
+The SORA Builders Programme process is divided in four sections, **Application**, **Review**, **Preparation**, and **Product Launch**;
+	
+1. **Application** - After you have submitted your proposal, it will be reviewed
+internally and you will be invited to talk to the developers, if it is a match, the process will continue. **48 hr**.
+	
+2. **Review** - A review process with an NDA and Memorandum of Understanding (MoU) to begin building will take place. This will result in a formal agreement to become a SORA Builder. This process takes approximately **one month**.
+	
+3. **Preparation** - The builder submits an RFP which contains all the basic information about what is to be built and the milestones of development. An AMA will be held where the SORA community can ask the builders any questions they have regarding the project. If the community is satisfied with the answers, the governance proposal to fund the builders will begin. This process takes approximately **two months**.
```

Comment:
We should mention somehow that the funding step is reviewed case by case. Some builders could get funded before delivering, throughout the development, while other builders would only get funded once they delivered on the RFP / milestones
@WRRicht3r

Reply:
I added an info box mentioning this and some possible variables.